recipes/glib/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set major_minor = "2.48" %}
+{% set version = major_minor + ".0" %}
+{% set sha256 = "744be6931ca914c68af98dc38ff6b0cf8381d65e335060faddfbf04c17147c34" %}
```

Comment:
Yes, that is the common argument against it.  I think having it up to makes it more explicit that it is something that requires changes when the version changes.  I don't think it's a big deal either way, though.  Whatever the maintainers want to do is fine by me.